swaps/rebalance.js

```diff
@@ -610,9 +611,10 @@ module.exports = (args, cbk) => {
             return ![findInKey, findOutKey].includes(n.from_public_key);
           }),
           in_through: getInbound.public_key,
+          is_strict_max_fee: args.is_strict_max_fee,
           logger: args.logger,
           lnd: args.lnd,
-          max_fee: defaultMaxFeeTotal,
+          max_fee: args.max_fee || defaultMaxFee,
```

Comment:
```suggestion
          max_fee: args.max_fee ?? defaultMaxFee,
```

I think this would make it so that if you enter 0 for your max fee then it won't evaluate routes that aren't fee 0, as it is here if you put max fee as 0 it would use the default max fee instead

Reply:
There is an initial validation check for rebalance to now allow --max-fee or --max-fee-rate to be 0 to be zero I think, do you want to get rid of that?

Reply:
interesting, i think that can be left alone for now but I'm not sure why that's there, maybe it should be removed in the future

Reply:
Cool.